### PR TITLE
SSIS File System Support

### PR DESCRIPTION
**Pull request by HaoQian-MS**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5739 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5739/commits/d5e28c6bf69a6eb471530e44648e39f3d63be349

**Review by veronicagg — CHANGES_REQUESTED** [review]
Have the changes to these properties been deployed in the API already?

File: specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Pipeline.json [hunk 1]
```diff
@@ -2075,6 +2075,10 @@
           "additionalProperties": {
             "$ref": "#/definitions/SSISPropertyOverride"
           }
+        },
+        "logLocation": {
+          "description": "SSIS package exection log location.",
```
Comment (nschonni) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Pipeline.json]:
```suggestion
          "description": "SSIS package execution log location.",
```

File: specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Pipeline.json [hunk 2]
```diff
@@ -2155,6 +2194,68 @@
...
+        }
+      },
+      "required": [
+        "domain",
+        "userName",
+        "password"
+      ]
+    },
+    "SSISLogLocation": {
+      "description": "SSIS package exection log location",
```
Comment (nschonni) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Pipeline.json]:
```suggestion
      "description": "SSIS package execution log location",
```

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json [hunk 3]
```diff
@@ -2424,6 +2424,10 @@
           "additionalProperties": {
             "$ref": "#/definitions/SSISPropertyOverride"
           }
+        },
+        "logLocation": {
+          "description": "SSIS package exection log location.",
```
Comment (nschonni) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]:
```suggestion
          "description": "SSIS package execution log location.",
```

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json [hunk 4]
```diff
@@ -2504,6 +2543,68 @@
...
+        }
+      },
+      "required": [
+        "domain",
+        "userName",
+        "password"
+      ]
+    },
+    "SSISLogLocation": {
+      "description": "SSIS package exection log location",
```
Comment (nschonni) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json]:
```suggestion
      "description": "SSIS package execution log location",
```

**Comment by veronicagg** [comment]
@HaoQian-MS please let me know if the properties have been deployed, if so I can approve and merge this PR. 
Please resolve conflicts.

**Comment by HaoQian-MS** [comment]
@veronicagg , we decided to hold this change currently as some adjustment is needed for our feature.

**Comment by veronicagg** [comment]
@HaoQian-MS ok, thanks for letting me know. Should we close the PR until you're ready to come back to it?

**Comment by HaoQian-MS** [comment]
@veronicagg  , if it's closed, can I reopen it later? If not, can we keep it open?

**Comment by veronicagg** [comment]
@HaoQian-MS yes, we can reopen it at anytime, just let me know if you're not able to reopen it. Thanks!

**Pull request closed without merging**